go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bank-vaults/vault-secrets-webhook
 
-go 1.21.1
+go 1.21
```

Comment:
`go 1.21` is not a valid go toolchain/tag (https://github.com/golang/go/tags); `go 1.21.0` is